runtime/parser/parse_metrics_view.go

```diff
@@ -63,6 +63,15 @@ type MetricsViewYAML struct {
 		ValidPercentOfTotal bool           `yaml:"valid_percent_of_total"`
 		TreatNullsAs        string         `yaml:"treat_nulls_as"`
 	}
+	Annotations []*struct {
+		Name           string             `yaml:"name"`
+		Model          string             `yaml:"model"`
+		Database       string             `yaml:"database"`
+		DatabaseSchema string             `yaml:"database_schema"`
+		Table          string             `yaml:"table"`
+		Connector      string             `yaml:"connector"`
+		Measures       *FieldSelectorYAML `yaml:"measures"`
+	} `yaml:"annotations"`
```

Comment:
These new properties also need to be documented in `rill/runtime/parser/schema/project.schema.yaml`